build_container.sh

```diff
@@ -5,6 +5,11 @@ set -e
 NPM_TAG=$1
 IMAGE_NAME=$2
 NODEVERSION=$3
+PORT=${4:-3000}
+shift
```

Comment:
@dealfonso do you mind explaining what you are attempting to perform here or clean up?
I think in the long run it might be good to document the lines so it'd be easier to maintain and understand in the future.

Reply:
I'm fine with the logic of it :)
Just wanted to see if we could add some documentation so it'd be easier to follow for future references.

Reply:
@vince-fugnitto do you need any extra work from my side?

Reply:
@dealfonso only if you could provide some comments or documentation in the `build_container.sh` regarding the `shift`? It is not critical, but it will help future developers and also external contributors in better understanding what is going on and why it is needed. Then I think the PR is ready :)

Reply:
@dealfonso I think your excellent explanation above would work nicely if you could add it to the script.

@vince-fugnitto WDYT?

Reply:
Hi @marcdumais-work and @vince-fugnitto,

I have just included the comments in the script.